.gitignore

```diff
@@ -1,5 +1,9 @@
-/build
-/lake-packages
-/.cache
```

Comment:
When did we stop using `.cache`?

Reply:
`.cache` is still used, when the user does not have a home directory as detected via env vars.

Reply:
I added it back along with a speculative comment